packages/babylon/CHANGELOG.md

```diff
@@ -620,7 +620,7 @@ function *foo() {
 Annotate more errors with expected token ([#172](https://github.com/babel/babylon/pull/172))) (Moti Zilberman)
 
 ```js
-// Unexpected token, expected ; (1:6)
+// Unexpected token, expected ";" (1:6)
```

Comment:
I think in that the CHANGELOG shouldn't be updated, since in the version referred by this section the error message was without the double quotes.

Reply:
Cool, yeah, I wasn't sure about that. I'll revert it.